script.js

```diff
@@ -22,3 +22,28 @@ function mobileMenu() {
 }
 
 mobileMenu();
+
+const formStorage = document.querySelector('#form1');
+const nameTxt = formStorage.querySelector('#name1');
+const emailTxt = formStorage.querySelector('#email1');
+const msgTxt = formStorage.querySelector('#msg');
+
+const saveData = () => {
+  const dataList = {
+    name: nameTxt.value,
+    email: emailTxt.value,
+    msg: msgTxt.value,
+  };
+  localStorage.setItem('dataList', JSON.stringify(dataList));
+};
+
+nameTxt.addEventListener('change', saveData);
+emailTxt.addEventListener('change', saveData);
+msgTxt.addEventListener('change', saveData);
```

Comment:
- [Optional] While this way works, it unfortunately only saves on focusing out of the element on chromium-based browsers, which have the largest market share, it'd be better to use 'input' instead of 'change', to avoid a situation when someone accidentally refreshes a page in the middle of typing a long message in the text area